usagov/usagov-2021
# Usagov 1731 preload largest contentful paint

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1731

## Description


Preload of the main usagov banner image.

## Type of Changes

  - Add screenshot showing what it should look like
- [x ] Other


## Testing Instructions



1. Open chrome dev tools
2. Click the network tab
3. Refresh the page
4. In the Network tab, check that the Homepage_banner.png is the first thing downloaded
5. Repeat these steps for all the screen sizes 
<img width="455" alt="image" src="https://github.com/user-attachments/assets/827692c0-fe57-43c9-a6af-fa1ec594e36c">

6. Make sure there are no errors in the console.

### Change Requirements



### Validation Steps

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
8. Build and deploy the changes.
9. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
